src/pybind/mgr/dashboard/frontend/src/app/core/navigation/identity/identity.component.html

```diff
@@ -3,7 +3,7 @@
   <li disabled="true"
       class="show cds--overflow-menu-options__option cds--overflow-menu-options__option--disabled my-2"
       i18n>
-    <div class="cds--overflow-menu-options__btn">Logged in as {{ username }}
+    <div class="cds--overflow-menu-options__btn">Logged in as {{ username.length > 5 ? username.slice(0,4) + '...' : username }}
```

Comment:
I saw we have a `truncate` [pipe](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/truncate.pipe.ts) which might be doing the same thing? Have you tried it out?